next-env.d.ts

```diff
@@ -0,0 +1,5 @@
+/// <reference types="next" />
+/// <reference types="next/image-types/global" />
+
+// NOTE: This file should not be edited
```

Comment:
Do we need to commit this file? Is it not added during the build?

Reply:
I think it stays here. Cf inspect: https://github.com/lukso-network/tools-erc725-inspect/blob/main/next-env.d.ts

Reply:
actually it should be part of the gitignore but it is not added automatically. Will add it manually